tests/Polyfill/EmptyEscapeParserTest.php

```diff
@@ -124,6 +124,35 @@ public function testPreserveEmptyLines()
         }
     }
 
+    /**
+     * @covers ::parse
+     * @covers ::extractRecord
+     * @covers ::extractFieldContent
+     * @covers ::extractEnclosedFieldContent
+     */
+    public function testReadingOnlyStream()
+    {
+        $source = <<<EOF
+"1","2"
+
+EOF;
+
+        $expected = [
+            ['1', '2'],
+            [null],
+        ];
+
+        $path = sys_get_temp_dir().'/test.csv';
+        file_put_contents($path, $source);
```

Comment:
Is it possible to use `php://temp` or add a file to the `data` directory under so that tests are not dependent of the underlying OS

Reply:
I completely overlooked the `data` directory. I've rewritten the test to use the `foo_readonly.csv` file.